staging/src/k8s.io/client-go/util/clientutil/clientutil.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package keyutil contains utilities for managing public/private key pairs.
+package clientutil
+
+import (
+	"fmt"
+	clientset "k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func CreateClientFromKubeconfigFile(kubeconfigPath string) (clientset.Interface, error) {
+
+	clientConfig, err := CreateClientConfigFromKubeconfigFile(kubeconfigPath)
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := clientset.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, fmt.Errorf("error while creating clientset with %s, error %v", kubeconfigPath, err.Error())
+	}
+
+	return client, nil
+}
+
+func CreateClientConfigFromKubeconfigFile(kubeconfigPath string) (*restclient.Config, error) {
+	return CreateClientConfigFromKubeconfigFileAndSetQps(kubeconfigPath, 0, 0, "")
+}
+
+func CreateClientConfigFromKubeconfigFileAndSetQps(kubeconfigPath string, qps float32, burst int, contentType string) (*restclient.Config, error) {
```

Comment:
I am debating whether QPS and burst shall be passed into this function.

Reply:
Can you add some comment for this function to explain qps and burst is not required?

Reply:
it is a convenient way for functions in hollow-node.go to get the client config. for function not need the QPS set, they call the other function for only passing inthe kubeconfigs. this is internal, not a public API btw.

Reply:
The fact that this function is now in client-go and named started with uppercase make it an public API. Adding comment makes it easier to be understood later.

Reply:
those are NOT intended to be public API at all. if adding to the client-go is not a good way. we shall put it else-where to avoid such confusions. some comment can help so i will add the comments there.